clarity_ext/service/dilution/service.py

```diff
@@ -122,13 +122,19 @@ def evaluate_transfer_route(self, transfer, transfer_handlers):
     def set_max_destination_volume(self, transfers, robotsettings):
         dest_type = self._get_destination_container_type(transfers)
         if dest_type == Container.CONTAINER_TYPE_TUBE:
-            self.max_pipette_vol_for_row_split = robotsettings.max_pipette_vol_for_row_split_tube
+            self.max_destination_volume = robotsettings.max_destination_volume_tube
         else:
-            self.max_pipette_vol_for_row_split = robotsettings.max_pipette_vol_for_row_split_plate
+            self.max_destination_volume = robotsettings.max_destination_volume_plate
 
     def _get_destination_container_type(self, transfers):
         types = list(set([t.target_location.artifact.container.container_type for t in transfers]))
-        return types[0]
+        return utils.single(types)
+
+    def _sorted_transfers(self, original_transfers):
+        sort_strategy = self.dilution_settings.tube_placement_sort_strategy
```

Comment:
I wonder if `tube_placement_sort_strategy` shouldn't have a more generic name (perhaps just sort_strategy) since it's being addressed in code that will work for tubes as well as plates.

Reply:
I wanted to distinguish the sort strategy for placing tubes from the sort strategy for sorting the robotfile, that is used later in the dilution code. (robotfile sorting could not be used here because all handlers must be exectued first). I wonder if it could be okay to keep the tube-placement-sort-strategy? When configuring the dilution-settings, I think its fairly explicit what it's about.

Reply:
Ahh I see!